bin/check-sidekiq-latency.rb

```diff
@@ -59,7 +59,7 @@ class SidekiqLatencyCheck < Sensu::Plugin::Check::CLI
    def run
      begin
        Sidekiq.configure_client do |sidekiq_config|
-        sidekiq_config.redis = { url: config[:redis_url], timeout: 20 } # timeout in seconds
+        sidekiq_config.redis = { url: config[:redis_url], timeout: 60 } # timeout in seconds
```

Comment:
I know it’s from https://chartmogul.slack.com/archives/C075XDD0X/p1559304579008600
But couldn’t tell from this PR description.